### PR TITLE
Added - Vehicle AT Ammo

### LINKED ISSUE #20: NAF - Vehicle Inventories to include some MAAWS rounds
> Adding some MAAWS rockets into vics like MRAPS and APCs would make life much easier for AT in many OPs, especially when they're in a smaller section without an assistant
> And restricting it to MRAP/APCs would ensure that we don't have a complete overflow of rockets (i.e. replacing ammo-carriers)
> I'm thinking maybe 5 HEAT 75 / 1 HE (per vehicle)

-- [Klonke](https://discord.com/channels/532683310409842728/733501366626025604/812755503498526720)

___

Missing some MAAWs rounds!

### PR DESCRIPTION
**Pull request by See0ne**
Added AT ammo combinations of 1 HEAT/1HE or 3 HEAT/1 HE into a number of NAF vehicles as per suggestion from Klonke and issue #20 
Also updated Readme's list of dependencies to remove references to mods removed in v4.0

close #20

### CODE REVIEW

File: addons/norwegian_armed_forces/vehicles/index.hpp [hunk 1]
```diff
@@ -65,8 +65,30 @@ class _xx_ACE_tourniquet { \
...
+class _xx_MRAWS_HE_F { \
+	name = "MRAWS_HE_F"; \
+	count = 1; \
+}; \
+class _xx_MRAWS_HEAT_F { \
+	name = "MRAWS_HEAT_F"; \
+	count = 1; \
+};
+
+#define VEHICLE_AT_AMMO_EXTE \
```
Comment (Saborknight) [inline addons/norwegian_armed_forces/vehicles/index.hpp:80]: Can we have a name that makes more sense here? Is this intended for armoured or what?

**Review by Saborknight — COMMENTED** [review]
Is it necessary for the MAAWS rounds to be placed in their own defined thing (momentarily forgotten the name)? Why can't they be added to the existing ones for the Armoured and Non-Armoured vehicles?

File: addons/norwegian_armed_forces/vehicles/index.hpp (hunk 1 above)
Comment (See0ne) [reply addons/norwegian_armed_forces/vehicles/index.hpp:80]: EXTE was put in as shorthand for extended. I can put it in as the full word though

Comment (Saborknight) [reply addons/norwegian_armed_forces/vehicles/index.hpp:80]: Aye, probably best

**Comment by See0ne** [comment]
The MAAWS ammo was defined under its own thing because it was being added to not all vehicles strictly armoured and unarmoured. (Not being added to tanks for example, as well as some helicopters getting the base ammo and others getting the full ammo). The commit lists which vehicles received which values but this seemed to be the easier way to handle the assigning of specific vehicles

**Comment by Saborknight** [comment]
Makes sense.

**Pull request merged**